### ISSUE TITLE
잡담

### ISSUE THREAD

**Issue by Lee-SungKyu** [Lee-SungKyu/OSSteam14#5]

**Comment by Lee-SungKyu** [comment]
그 코딩 해더 파일앞에 이니셜 붙여주세요! 그러면 정리하기가 편해서요 ㅎㅎ 예를들면 이성규니까 Lproduct.h

**Comment by Lee-SungKyu** [comment]
Struct도 만들어야되서 제가 빨리 만들어서 여기다 올릴게요!

**Comment by Lee-SungKyu** [comment]
//text1.file
typedef struct
{
        int room_number;
        int bed_count;
        int head_count;
        int check;
}Hotel;


//text2.file
typedef struct
{
        int room_number2;
        int head_count2;
        char name2[100];
}Reserve;

**Comment by Lee-SungKyu** [comment]
제가 struct 만들어놨으니까 가능한 빨리 역할 분담된 코딩 해주시고 PR 보내주시면 감사하겠습니다! 포인터 배열로 코딩할 생각인데 괜찮으신가요?

**Comment by Lee-SungKyu** [comment]
시간이 그렇게 많지 않은 관계로 제가 Save/Load까지 구현하겠습니다. 나머지 만드셔서 PR 보내주세요~! 민석 학우님이 CRUD을 끝내셔야 CRUD 외 함수 코딩 실행이 가능해요! :)

**Comment by LazyBear8372** [comment]
데이타 삭제하는 함수 말인데요 삭제된 데이타는 room_number 변수값에 -1 값을 넣어서 삭제됬다는 것을 표시하는거 어떤가요?

**Comment by Lee-SungKyu** [comment]
head_count 를 -1로 하시면 됩니다! PR 확인했구요 조금만 있다가 수정할게요~

**Comment by LazyBear8372** [comment]
이번에 PR 들어간거 받아주시면 마지막으로 PR 하나만 더 보낼게요!

**Comment by Lee-SungKyu** [comment]
네~

**Comment by Lee-SungKyu** [comment]
reserve.txt 을 load 성공했는데 hotel.txt에 문제가 조금있어서 고치고 있을게요~

**Comment by LazyBear8372** [comment]
저희 오늘까지 wiki 작성해야 하는데 짧게 wiki 하나만 만들어주실 수 있나요? 아무래도 저는 wiki를 만들 권한이 없는거같아서요
그리고 과제제출 하셨나요? 과제제출은 읽어보니까 개인적으로 제출해야 하나봐요.
URL만 복사 붙여넣기 하면 되니까 아직 제출 안하셨으면 하는게 좋을거같아요!!

**Comment by Lee-SungKyu** [comment]
네 저번시간에 일단 기본 WIKI 틀은 만들어 놨고요 아직 제출 안했습니다! 다 끝나면 여기 끝났다고 말씀드리고 URL 제출할게요 :)

**Comment by Lee-SungKyu** [comment]
모든 기능 전부다 작동되게 코딩 끝났고요. Hotel.txt 보시면 호 수 따라 최대 인원 수 (Head_count)가 있어요 그걸 Create + Update 떄 방 번호를 선택했을때 따라 인원 수를 입력받았을떄 초과하면 다시 입력받도록 코딩 부탁드립니다, 더 이해하기 쉽도록 Create 랑 Update에 방 호 수 선택할때 while 써서 101~105  | 201~205 .. etc 안에 선택하도록 해놨습니다.

**Comment by Lee-SungKyu** [comment]
다 끝나시면 이제 실행 결과 사진찍어서 WIKI 작성하면 끝날거 같아요 ㅎㅎ :)
